tests/src/packages/websocket/WebSocketTests.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package packages.websocket
+
+import java.net.URI
+
+import org.junit.runner.RunWith
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.Finders
+import org.scalatest.junit.JUnitRunner
+
+import common.TestHelpers
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+import spray.json.pimpAny
+
+@RunWith(classOf[JUnitRunner])
+class WebSocketTests
+        extends TestHelpers
+        with WskTestHelpers
+        with BeforeAndAfterAll {
+
+    implicit val wskprops = WskProps()
+    val wsk = new Wsk()
+    var serverURI: URI = null
+
+    val websocketSendAction = "/whisk.system/websocket/send"
+
+    behavior of "Websocket action"
+
+    override def beforeAll() {
+        serverURI = WebSocketUtils.startWebSocketServer()
```

Comment:
you could replace `var serverURI: URI = null` with `val serverURI = WebSocketUtils.startWebSocketServer()` and remove `beforeAll`.